Subject: [PATCH] guid.c: include stdlib.h for bsearch

Implicit declaration; breaks when headers are not pulled in
transitively (gcc-14+/glibc-2.4x).
--- a/src/guid.c	2024-01-31 23:08:46.000000000 +0300
+++ b/src/guid.c	2026-10-22 16:52:38.061960804 +0300
@@ -9,6 +9,7 @@
 #include <dlfcn.h>
 #include <errno.h>
 #include <stdio.h>
+#include <stdlib.h>
 
 #include "efivar.h"
 
